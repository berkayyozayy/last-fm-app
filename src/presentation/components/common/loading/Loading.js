import React from "react";
import classes from "./Loading.module.css";

function Loading() {
  return (
    <div className={classes.loading}>
      <div className={classes.circle}></div>
    </div>
  );
}

export default Loading;
