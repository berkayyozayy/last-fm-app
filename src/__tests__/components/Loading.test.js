import { render } from "@testing-library/react";
import Loading from "../../presentation/components/common/loading/Loading";

describe("loading component", () => {
  test("it should render", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
