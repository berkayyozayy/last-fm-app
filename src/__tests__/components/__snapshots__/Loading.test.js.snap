// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading component it should render 1`] = `
<div>
  <div
    class="loading"
  >
    <div
      class="circle"
    />
  </div>
</div>
`;
